### ISSUE TITLE
Peer dependency: next@"^13.4"

### ISSUE THREAD

**Issue by andreisocaciu** [47ng/next-usequerystate#380]
Now that [NextJS@14.0.0](https://github.com/vercel/next.js/releases/tag/v14.0.0) is out, the peer dependency fails to install.

```
npm ERR! code ERESOLVE
--
15:22:04.041 | npm ERR! ERESOLVE could not resolve
15:22:04.042 | npm ERR!
15:22:04.042 | npm ERR! While resolving: next-usequerystate@1.8.4
15:22:04.042 | npm ERR! Found: next@14.0.0
15:22:04.042 | npm ERR! node_modules/next
15:22:04.042 | npm ERR!   next@"14.0.0" from the root project
15:22:04.042 | npm ERR!   peer next@">= 12.0.0" from next-i18next@13.2.2
15:22:04.042 | npm ERR!   node_modules/next-i18next
15:22:04.043 | npm ERR!     next-i18next@"13.2.2" from the root project
15:22:04.043 | npm ERR!
15:22:04.043 | npm ERR! Could not resolve dependency:
15:22:04.043 | npm ERR! peer next@"^13.4" from next-usequerystate@1.8.4
15:22:04.043 | npm ERR! node_modules/next-usequerystate
15:22:04.043 | npm ERR!   next-usequerystate@"1.8.4" from the root project
15:22:04.043 | npm ERR!
15:22:04.043 | npm ERR! Conflicting peer dependency: next@13.5.6
15:22:04.043 | npm ERR! node_modules/next
15:22:04.043 | npm ERR!   peer next@"^13.4" from next-usequerystate@1.8.4
15:22:04.043 | npm ERR!   node_modules/next-usequerystate
15:22:04.043 | npm ERR!     next-usequerystate@"1.8.4" from the root project
15:22:04.043 | npm ERR!
15:22:04.043 | npm ERR! Fix the upstream dependency conflict, or retry
15:22:04.043 | npm ERR! this command with --force or --legacy-peer-deps
15:22:04.044 | npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by franky47** [comment]
I published 1.9.1 yesterday which should have the correct peerDeps for Next.js 13.4+ and 14.

**Comment by andreisocaciu** [comment]
Fixed.
